packages/datadog-instrumentations/src/aws-sdk.js

```diff
@@ -167,7 +167,8 @@ function getChannelSuffix (name) {
     'sns',
     'sqs',
     'states',
-    'stepfunctions'
+    'stepfunctions',
+    'bedrock runtime'
```

Comment:
nit: we can just have this check `bedrock`, since `bedrock runtime` includes `bedrock`

Reply:
Actually, disregard. I don't think I'm right here haha

Reply:
It does feel a little naughty to use a space in the channel name though I suppose there is no technical reason not to.